src/Compilers/CSharp/Portable/Symbols/MissingModuleSymbol.cs

```diff
@@ -185,6 +185,8 @@ internal override bool HasAssemblyRuntimeCompatibilityAttribute
         }
 
         public override ModuleMetadata GetMetadata() => null;
+
+        public override bool? NonNullTypes => false;
```

Comment:
>NonNullTypes => false; [](start = 30, length = 22)

Do we rely on the specific value? `null` seems a better default

Reply:
> Do we rely on the specific value? null seems a better default

This API is going away. At this point any value is better than a crash in implementation from base.

---
In reply to: [223880050](https://github.com/dotnet/roslyn/pull/30387#discussion_r223880050) [](ancestors = 223880050)